src/dataflow/components/dataflow-program-graph.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react";
+import { Line } from "react-chartjs-2";
+import { ChartOptions, ChartData, ChartDataSets } from "chart.js";
+import { MAX_NODE_VALUES } from "./dataflow-program";
+import { NodePlotColors } from "./../utilities/node";
+import "./dataflow-program-graph.sass";
+
+export interface DataPoint {
+    x: number;
+    y: number;
+}
+export interface DataSequence {
+  name: string;
+  units: string;
+  data: DataPoint[];
+}
+export interface DataSet {
+  sequences: DataSequence[];
+}
```

Comment:
This notion of a `DataSet` contains an array of `DataSequences` and each sequence contains information about specific data recorded from a node including the x,y pairs as `DataPoints` is recycled from how data was maintained in DF2.  Note `x` and `y` could be `time` and `value` if we want data to be more closely coupled with how Dataflow creates data.

Reply:
This makes sense.